pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ilc-scraper"
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
This should not be a part of this PR. I'll bump the scraper version separately. One PR should only handle one change. Consider what would happen if we had to revert this change - we still wouldn't want to rescind the version change for the scraper.

Reply:
Done.